### ISSUE TITLE
Autotry mode

### ISSUE THREAD

**Issue by matklad** [bors-ng/bors-ng#215]
It would be useful if bors could be configured to automatically `try` a pull request on every PR update. That way the contributor gets faster feedback about the tests, as it may be inconvenient to run the whole testsuite locally. 

To some degree, this problem is solved by Travis, which automatically builds the pull requests, but currently this results in quite a few unnecessary builds. If someone sends a PR, and I r+ it, then travis builds PR twice (pr branch itself and the merge commit) and one more build is done by bors. If everything was handled by bors, then there would be only one build! Moreover, the r+ might be able to reuse the results of the try if the state of master didn't change since.

**Comment by notriddle** [comment]
Would #185 also be a reasonable fix for this?

**Comment by matklad** [comment]
Yeah, #185 will certainly help here, but it won't be a perfect solution: travis always builds PR's twice, while in some cases it is sufficiently to build only the merge commit. In general, looks like bors has a more complete picture about the status of the PRs, master, and queue, so potentially it can schedule CI's more efficiently, though this might be out of scope.
